test/Psr/CacheItemPool/CacheItemPoolDecoratorTest.php

```diff
@@ -1052,4 +1053,22 @@ public function testKeepsDeferredItemsWhenCommitFails(): void
         self::assertTrue($this->adapter->hasItem('keyOfFailedItem1'));
         self::assertTrue($this->adapter->hasItem('keyOfFailedItem2'));
     }
+
+    public function testPassesClockToCacheItem(): void
+    {
+        $clock   = $this->createMock(Clock::class);
```

Comment:
We should not mock a clock but instead use the corresponding `FrozenClock` instead

Reply:
```suggestion
        $now = new DateTimeImmutable();
        $clock   = new FrozenClock($now);
```

Reply:
I do explicitly want to ensure that the method is called. Thats why I prefer the mock here. We can use frozen clock in other scenarios for sure but since I removed `lcobucci/clock` in favor of `stella-maris` which does not provide such a frozen clock, I guess I keep working with the mock for now.